From acb7e8f7448efef4ba1d86247cacbd201df733ab Mon Sep 17 00:00:00 2001
From: Christopher Bostic <cbostic@linux.vnet.ibm.com>
Date: Tue, 6 Jun 2017 16:08:59 -0500
Subject: [PATCH] drivers/fsi: Add module license to core driver
Git-commit: acb7e8f7448efef4ba1d86247cacbd201df733ab
Patch-mainline: v4.13-rc1
References: bsc#1085115

Add missing MODULE_LICENSE("GPL") to the core FSI driver.

Signed-off-by: Christopher Bostic <cbostic@linux.vnet.ibm.com>
Signed-off-by: Greg Kroah-Hartman <gregkh@linuxfoundation.org>
Acked-by: Takashi Iwai <tiwai@suse.de>

---
 drivers/fsi/fsi-core.c |    1 +
 1 file changed, 1 insertion(+)

--- a/drivers/fsi/fsi-core.c
+++ b/drivers/fsi/fsi-core.c
@@ -57,3 +57,4 @@ static void fsi_exit(void)
 
 module_init(fsi_init);
 module_exit(fsi_exit);
+MODULE_LICENSE("GPL");
